### DISCUSSION TITLE
Migrating peers from Windows 11 to MacOS Sonoma

### DISCUSSION THREAD

**Discussion by gilvandev** [rustdesk/rustdesk#8132] [General]
### Bug Description

When I've tried to copy my peers from Windows 11 'AppData/Roaming/RustDesk/peers' to MacOS Sonoma '/Users/MyUser/Library/com.carriez.RustDesk/peers'... reload the app on MacOS get stuck on "Generating..." on ID field and no peers shows on the list

### How to Reproduce

Copy the peers files from a Windows machinhe/vm to other machine with MacOS Sonoma (14.5)

### Expected Behavior

Show the peers list on flutter UI like a normal history of connection

### Operating system(s) on local side and remote side

Windows 11 -> macOS Sonoma (14.5)

### RustDesk Version(s) on local side and remote side

1.2.4 -> 1.2.4

### Screenshots

![Captura de Tela 2024-05-23 às 21 02 43](https://github.com/rustdesk/rustdesk/assets/671468/42d97c75-c8fe-4bb7-afe2-3a2824bf74d9)
![Captura de Tela 2024-05-23 às 21 01 57](https://github.com/rustdesk/rustdesk/assets/671468/6c13718c-d492-427e-b4cb-224152e96640)
![Captura de Tela 2024-05-23 às 21 04 36](https://github.com/rustdesk/rustdesk/assets/671468/2c10894e-4a2e-46df-9f0d-3cf7d1feb594)

### Additional Context

Tried too with RustDesk versions 1.2.4 to 1.2.3-2

**Comment by xlionjuan** [comment]
I'm guessing CRLF and LF problems

**Reply by gilvandev** [reply]
Maybe... How can I test this? Editing the toml file in the Mac with VS Code?

**Reply by xlionjuan** [reply]
You could try it

**Reply by gilvandev** [reply]
I´ve tried and success! Edited the toml files with VS Code and delete the spaces between lines and make new lines with a simple ENTER on the end of each one.

But now: how this could be fixed without manual editing? I have at least 50 peers and maybe other people have too or much more...

**Reply by xlionjuan** [reply]
I think this kind of tool exists, but I don't know where to find

Or maybe git with `.gitattributes` may be a solution, just put this file and those files to your **private** git repo, push them and pull on your another computer.

https://github.com/rustdesk/rustdesk/blob/9562768a048f0dccfc88cf8e214c62e919193487/.gitattributes#L1

**Comment by gilvandev** [comment]
But there's still a problem. I believe that editing a file created the illusion that the problem had been solved. However, trying with other files, I was unsuccessful.

I noticed in VS Code and Notepad (Windows) that the encoding is already in UTF-8 and with LF

**Comment by gilvandev** [comment]
Hi, I discovered the real problem. Saved passwords on the toml file. Remove node "password" from all the files and works fine. Maybe something about the password's cipher between the different OS or machine

Tested with the new 1.2.6 version
